src/chapters/booleans-and-conditionals/booleans.rst

```diff
@@ -0,0 +1,209 @@
+.. _booleans:
+
+Booleans
+========
+
+.. index:: data type
+
+One of the core features of any programming language is the ability to conditionally execute a segment of code. This means that a program will run a segment of code *only if* a given condition is met. Another way to say this is that the segment of code only if the program is in a particular state.
+
+.. admonition:: Example
+
+   Consider a banking application that can remind you when a bill is due. The application will notify the user that a bill is due soon, but *only if* the bill has not already been paid.
+
+In this example, we can phrase the condition as follows: Send a notification of an upcoming bill if the statement "the bill is unpaid" is true. In order to formalize such a statement in code, we need a notion of what it means to be true or false.
+
+Boolean Values
+--------------
+
+.. index:: ! true, ! false, ! boolean
+
+.. index::
+   single: boolean; value
+
+The JavaScript data type for storing true and false values is ``boolean``, named after the British mathematician, George Boole. George Boole created `Boolean Algebra <https://en.wikipedia.org/wiki/Boolean_algebra>`_ which is the basis of all modern computer arithmetic.
+
+There are only two **boolean values**. They are ``true`` and ``false``. As always, JavaScript is case-sensitive, so ``True`` and ``False`` are not valid boolean values.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(true);
+      console.log(typeof true);
+      console.log(typeof false);
+
+   **Output**
+
+   ::
+
+      true
+      boolean
+      boolean
+
+It is extremely important to realize that ``true`` and ``false`` are **not** strings. Surrounding the words "true" and "false" in quotes in JavaScript will result in string values rather than boolean values. 
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(typeof true);
+      console.log(typeof "true");
+
+   **Output**
+
+   ::
+
+      boolean
+      string
+
+Boolean Conversion
+------------------
+
+.. index:: ! Boolean(), type conversion
+
+As with the number and string data types, the boolean type has a type conversion function, ``Boolean()``. It works similarly to ``Number()`` and ``String()``, attempting to convert a non-boolean value to a boolean.
+
+.. admonition:: Try It!
+
+   Explore how ``Boolean()`` converts various non-boolean values.
+
+   .. sourcecode:: js
+
+      console.log(Boolean("true"));
+      console.log(Boolean("TRUE"));
+      console.log(Boolean(0));
+      console.log(Boolean(1));
+      console.log(Boolean(''));
+      console.log(Boolean('LaunchCode'));
+
+   `Run this program at repl.it <https://repl.it/@launchcode/Boolean-Type-Conversion>`_
+
+   Under which conditions does ``Boolean()`` convert a string to ``true``?
```

Comment:
This question seems to be a concept check, which should be outside of the 'Try It' box.

Reply:
I placed this in a Question admonition.